rllib/connectors/tests/test_agent.py

```diff
@@ -470,11 +469,20 @@ def test_connector_pipline_with_view_requirement(self):
         )
 
         env = gym.make("CartPole-v1")
-        policy = PPOTorchPolicy(
-            observation_space=env.observation_space,
-            action_space=env.action_space,
-            config=config.to_dict(),
-        )
+        policy = config.build().get_policy()
+
+        REQUIRED_KEYS = {
```

Comment:
I could not just pass in the entire view-requirements because only a subset of the keys will be passed in and will result in a key error failure. Filtering VRs to only those that are required fixes the issues.